api-client/src/models/OfferModel.ts

```diff
@@ -3,14 +3,14 @@ export default class OfferModel {
   _title: string
   _path: string
   _thumbnail: string
-  _postData: Map<string, string> | null
```

Comment:
Why is null better than undefined?

Reply:
Not quite sure what you are trying to say here ^^
I replaced `null` with `undefined`, as it can also be more convenient, as you can leave the parameter, instead of explicitly setting it to `null`. Furthermore `undefined` is more convenient for react, as there the properties are always set or `undefined`

Reply:
oke thank you. (i was somehow confused and thought you did it the other way round)